main.js

```diff
@@ -64,6 +64,7 @@ var USAGE           = "Error missing args. \n" +
     "--timeout `MSECS` : (optional) time in millisecs to wait for tests to pass|fail \n" +
                 "\t(defaults to 10 minutes) \n" +
     "--useTunnel: (optional) use tunneling instead of local address. default is false\n" +
+    "--args: (optional) add command line args to the \"cordova build\" and \"cordov run\" commands \n" +
```

Comment:
Just noticed this list is sorted alphabetically, so this should be a lot higher.